src/components/views/Registration.js

```diff
@@ -67,6 +67,9 @@ const Registration = () => {
       localStorage.setItem('token', response.headers.token);
       localStorage.setItem('id', response.data.id);
 
+      // do the login (especially change the status to online)
+      await api.post('/users/login', JSON.stringify({username, password}))
```

Comment:
This is what I changed (goal of this PR) but already merged the bugfix from the API into this PR